components/webdriver_server/lib.rs

```diff
@@ -715,13 +715,14 @@ impl Handler {
     }
 
     /// Send command to Script Thread with session's current browsing context.
-    /// If CHECK_OPEN is true, it would verify the existence of browsing context before sending.
-    fn browsing_context_script_command<const CHECK_OPEN: bool>(
+    /// If verify is Yes, it would verify the existence of browsing context before sending.
```

Comment:
```suggestion
    /// If `verify` is [`VerifyBrowsingContextIsOpen::Yes`], it would verify the existence of browsing 
    /// context before sending.
```